docs/Environment_Variables.md

```diff
@@ -0,0 +1,103 @@
+# Environment Variables (addition explaining the Values part)
+
+(this is supposed to go to the Environment book chapter)
+
+## Environment Variables Are Values
+
+Since Nushell extends the idea of classic shells "Everything is a text" into "Everything is a structured data", it feels natural to apply this philosophy to environment variables as well.
+In Nushell, environment variables can hold any value, not just a string.
+
+Since the host environment (i.e., the OS Nushell is running in) treats environment variables as strings, we need a way to convert them between strings and arbitrary values seamlessly.
+In general, there are two places when Nushell needs to interact with the host environment:
+1. On startup, Nushell inherits the host environment variables.
+2. When running an external program that is not part of Nushell's commands, the program expects environment variables to be strings.
+3. When an environment variable is passed to an extenal library in the Nushell's codebase (this includes plugins as well). These variables are listed later in this section.
+
+## Configuration
+
+By default, if you do not configure anything, all environment variables are imported as strings on startup and then directly passed to any external program we might be spawning.
+However, you can configure selected any environment variable to be converted to/from any value:
+
+```
+# config.nu
+
+let config = {
+    ... other config ...
+    env_conversions: {
+        FOO: {
+            from_string: {|s| $s | split row ':' }
```

Comment:
how about `(char seep)` instead of `:`

Reply:
I wanted to keep it simple, but maybe I should add a note there about these list-like env vars like PATH etc.